app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="post-section">
+  <% content_for :title, "#{@user.name}'s Posts" %>
+  <%= render "users/user", user: @user %>
+  <% @user.posts.each do | post | %>
+    <%= link_to user_post_path(@user, post) do %>
+      <%= render "post", post: post %>
+
+      <div>
+        <h4>
+          <span>Comments</span>
+          <span>for Post: </span>
+          <span><%="#{post.title}"%> </span>
+        </h4>
+        <div class='postComments'>
+          <% post.recent_comments.each do |comment| %>
+            <div>
+              <p>
+                <span><strong><%= comment.user.name%>:</strong></span>
+                <span><%= comment.text %></span>
+              </p>
+            </div>
+            <%if can? :destroy, comment%>
+              <%= button_to "Delete Comment",
+               user_post_comment(comment.author_id,
+                comment.post_id, comment), 
+                method: :delete, data: {turbo: false}%>
+          <% end %>
+        </div>
+      </div>
```

Comment:
As per your feedback in [click](https://github.com/Fahim-Khan-P/blog/pull/10#discussion_r1237043732), you can see that I've already implemented the `delete button` for the `comment` section!

Reply:
Hi @Fahim-Khan-P, the reason I suggested the change is that the delete button for the comment in the Post#index page doesn't work. It is the delete button for a post that works. 👍 😎